m/mex_tools/posixtime.m

```diff
@@ -51,9 +51,10 @@
 
   if ISOCTAVE
     t = time();
-  else
-    error('glider_toolbox:posixtime:MissingMexFile', ...
-          'Missing required mex file.');
+  else % -FC
+      t = (now()-datenum(1970,1,1))*86400;
```

Comment:
These changes should be reset (`git reset v1.1.1 m/mex_tools/posixtime.m`).
The required mex file can be build with `setupMexPosixtime`,
(it simply exposes the `time` function from the C standard library).
Even though the conversion here is correct, and you can use it if you do not want to compile the mex file,
it should *not* be part of this PR.